.idea/modules.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/cac-tripplanner-android.iml" filepath="$PROJECT_DIR$/cac-tripplanner-android.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
All the other `.idea` files were unmodified, but this one was dirtied by importing it into Android Studio, and should be added to `.gitignore`.

Reply:
Whoops, just saw this note after merge. Good to know, will add to `.gitignore` in a future PR.